### PR TITLE
feat(cli): add `app logs` command

### PR DESCRIPTION
**Pull request by jfeodor**
Adds the command `numerous app logs` which prints the app logs from a deployment of an app.

### CODE REVIEW

File: cli/cmd/app/logs/logs_test.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+func dummyPrinter(entry app.AppDeployLogEntry) {}
+
+func TestLogs(t *testing.T) {
+	const slug = "organization-slug"
+	const appName = "app-name"
+	testError := errors.New("test error")
+
+	t.Run("given invalid slug then it returns error", func(t *testing.T) {
+		appDir := t.TempDir()
+		test.CopyDir(t, "../../../testdata/streamlit_app", appDir)
```
Comment (moroderNumerous) [inline cli/cmd/app/logs/logs_test.go:24]: does the first two lines have any impact on the test? If not, delete them 😄  this also goes for the next test

File: shared/schema.gql [hunk 2]
```diff
@@ -340,6 +351,7 @@ extend type Subscription {
 ###############################################################################
 
 directive @canAccessSubscriptionOffer on FIELD_DEFINITION
+directive @canManageSubscription on FIELD_DEFINITION
```
Comment (moroderNumerous) [inline shared/schema.gql:354]: I assume the changes starting here and below, are related to something else, otherwise I don't understand it 😄

**Review by moroderNumerous — CHANGES_REQUESTED** [review]
Good stuff MVJ!

File: cli/cmd/app/logs/logs.go [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+				return nil
+			}
+			printer(entry)
+		case <-ctx.Done():
+			return nil
+		}
+	}
+}
+
+func getAppIdentifier(appDir string, slug string, appName string) (string, string, error) {
```
[the remark this answers is not in the document]
Comment (moroderNumerous) [reply cli/cmd/app/logs/logs.go:49]: I saw this is done in the PR with delete command, So i will resolve this 😄

File: cli/cmd/app/logs/logs_test.go (hunk 1 above)
Comment (jfeodor) [reply cli/cmd/app/logs/logs_test.go:24]: Yeah, I'll remove those :smile:

Comment (jfeodor) [reply cli/cmd/app/logs/logs_test.go]: In 81eaf91

**Pull request merged**